package.json

```diff
@@ -1,25 +1,16 @@
-########################################################################################################
-#   (c) Copyright 2015 Hewlett-Packard Development Company, L.P.
-#   All rights reserved. This program and the accompanying materials
-#   are made available under the terms of the Apache License v2.0 which accompany this distribution.
-#
-#   The Apache License is available at
-#   http://www.apache.org/licenses/LICENSE-2.0
-########################################################################################################
-
 {
-  "name": "atom-cloudslang-extension",
-  "version": "0.0.1",
+  "name": "language-cloudslang",
```

Comment:
package name and repo name can be different --look at https://github.com/dgerrells/atom-living-doc/blob/master/package.json -- similar to our repo